clang/lib/AST/PrintfFormatString.cpp

```diff
@@ -1111,6 +1116,9 @@ bool PrintfSpecifier::hasValidPrecision() const {
   case ConversionSpecifier::FreeBSDyArg:
   case ConversionSpecifier::PArg:
     return true;
+  case ConversionSpecifier::CHERIpArg:
```

Comment:
Maybe
```suggestion

  case ConversionSpecifier::CHERIpArg:
```
given the blank line below?